minetest/minetest
# DevTest: Add nodes and items for testing overlays

Fixes #12286.

For DevTest, tests for `overlay_tiles`, `inventory_overlay` and `wield_overlay` are added. Various combinations with the "global" `color` argument in the item definition are added as well. The nodes have one base tile and one overlay tile each. The craftitems were added to test the color set by item meta.

The tooltips describe what each item does.

## How to test

Read the code to see the item names, then give yourself these items in DevTest. Place them on the ground. Look at them in the inventory. Wield them. For the craftitems, one of them can be punched/placed to set a random color in item meta.

## Bug discovered during creation of this PR

* Wield image of the 2 craftitems remains uncolorized
